spec.md

```diff
@@ -179,6 +218,34 @@ physically present in the final circuit. Refer to the description of the
 instance statement for details on how to instantiate a module
 ([@sec:instances]).
 
+### Public Modules
+
+A public module is a module that is marked with the `public`{.firrtl} keyword.
+This means that this is a module that is usable external to this circuit.  A
+public module thereby has a defined ABI---knowing the FIRRTL representation is
+sufficient to know its compiled representation.
```

Comment:
```suggestion
A public module is a module that is marked with the `public`{.firrtl} keyword.
A public module has a defined ABI---knowing the FIRRTL representation is
sufficient to know its compiled representation.
This means that this is a module that is usable external to this circuit.  
```

Suggest switching the order of these things -- it's usable *because* it has a defined ABI.

Reply:
I'm also going to switch to saying "available" and not "usable" as the former language shows up in the LLVM linkage type that Will referenced.